### ISSUE TITLE
Implement ability to send Reconfigure message

### ISSUE THREAD

**Issue by tomaszmrugalski** [isc-projects/kea#84]
Sending a new message type is trivial. The tricky part will be to implement support for Authentication option. It must be able to store a digest, calculated from a key and message contents.

This will require implementing Authentication option. Note that option is used for two things. First, to send back the key value in plain text. Second, to send the digest.

**Comment by MayyaSunil** [comment]
Authentication option will be implemented separately. In this issue a new command for a single client and the mechanism for sending the message will be implemented.

**Comment by fxdupont** [comment]
A simple question: is there an available client with reconfigure and/or authentication support?
(I know none is supported by ISC DHCP)

**Comment by MayyaSunil** [comment]
From what I know currenty there are no clients supported for our testing :)

**Comment by fxdupont** [comment]
There is a rumor about dibber. Tomasz should answer when he will be back (public holiday in Poland).

**Comment by MayyaSunil** [comment]
Update: 
The implementation is currently on hold as unicast ip and keys required for sending the message is not available yet. This will be available only after issue #83.
 Other parts like command parser , packet packing is done on local branch. Pull request will be raised only after   issue #83 is integrated

**Comment by tomaszmrugalski** [comment]
This issue has been migrated to [gitlab #350](https://gitlab.isc.org/isc-projects/kea/issues/350).
